weaver/tests/network-setups/fabric/dev/scripts/setupCC.sh

```diff
@@ -1,11 +1,12 @@
 directory=$(dirname $0)
 
+CACTI_VERSION=v2@v2.0.0-alpha.1
```

Comment:
if you are putting `v2` to variable, then may be do it same for `line 9` as well?
Also instead of `v2@v2.0.0-alpha.1`, may be just store the `v2.0.0-alpha.1` as the version, and extract `v2` (first substring till the first `.`) for the first part, because that will be the convention for version like vA@A.B.C

Reply:
@VRamakrishna @sandeepnRES Just in case it is useful here: In other places in the code there's also examples of us using CloudFlare's bash semantic version parser in case you need to write logic based on major/minor/patch versions:

Usage:
https://github.com/hyperledger/cacti/blob/main/tools/docker/fabric-all-in-one/run-fabric-network.sh#L16

Where and how to get the .sh file to include: 
https://github.com/hyperledger/cacti/blob/main/tools/docker/fabric-all-in-one/Dockerfile_v2.x#L125

Reply:
@sandeepnRES I fixed the script as per your suggestion. Please take a look again.